src/main/docker/Dockerfile

```diff
@@ -1,8 +1,15 @@
 FROM frolvlad/alpine-oraclejdk8:slim
 
+EXPOSE 8000
```

Comment:
Ignore this line. I will remove before merging. This was added for remote java debugging.